SETUP/tests/phpunit_bootstrap.php

```diff
@@ -4,3 +4,4 @@
 include_once($relPath.'base.inc');
 include_once($relPath.'misc.inc');
 include_once($relPath.'wordcheck_engine.inc');
+include_once($relPath.'TableDocumentation.inc');
```

Comment:
Technically you don't need this, because the `dp_class_autoloader` will automagically load files for classes where the class name matches the filename. This is fine, but just FYI.

Reply:
Ah, that's useful, thanks for the info!